### PR TITLE
Bookstore Milestone 5 : Update to the UI styling

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## Hello reviewer(s) :wave: :wave: 

In this milestone I was able to:

- [x] Use the Zeplin information to improve the UI of the project
- [x] Document the project professionally by updating my README file
- [x] Deploy the project for testing purposes
- [x] Meet all the project milestone requirements
- [x] No linter error

Thank you for taking the time to review my project :pray:

### CODE REVIEW

File: src/stylesheets/Navbar.css [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
   width: 100%;
   height: 150px;
+  background-color: #fff;
 }
 
 .navbar nav {
   display: flex;
+  position: relative;
+
+  /* justify-content: space-around; */
```
Comment (julie-ify) [inline src/stylesheets/Navbar.css]: - [x] Kindly remove the commented code to achieve a cleaner codebase.
- [x] Please make your header i.e the name of your bookstore and the profile icon be contained within the book's card
- [x] Kindly reduce the size of the progress bar to match the Zeplin design

ApplicationFrameHost_wDlNn9HL07

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Nemwel-Boniface,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️  
- Good code organization ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/stylesheets/Navbar.css (hunk 1 above)
Comment (Nemwel-Boniface) [reply src/stylesheets/Navbar.css]: Thank you @julie-ify  for this review. I have made the requested changes

**Review by mikethreels — APPROVED** [review]
Hi Nemwel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**